docs/components/modeler/bpmn/signal-events/signal-events.md

```diff
@@ -21,6 +21,13 @@ Signal subscriptions only exists for the latest version of a process definition.
 process (based on the BPMN process id) will delete the old signal subscription. A new subscription is opened for the
 new deployed process definition.
 
+## Variable mappings
+
+When broadcasting a signal you can pass along variables. These variables are propagated to the process instance that's
+created and will be available at the process instance scope.
```

Comment:
❌ We need to expand this by explaining that the merging behavior can be customized using output mappings.

> When broadcasting a signal you can pass along variables. By default, all signal variables are merged into the process instance. This behavior can be customized by defining an output mapping at the signal catch event.